src/middlewared/middlewared/alembic/versions/22.12/2022-03-23_16-23_fix-nfs-paths.py

```diff
@@ -0,0 +1,47 @@
+"""Replace nfs_paths with nfs_path
+
+Revision ID: 696b3d876084
+Revises: 0b02e21a1a10
+Create Date: 2022-03-23 16:23:22.667861+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '696b3d876084'
+down_revision = '0b02e21a1a10'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('sharing_nfs_share', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('nfs_path', sa.TEXT(), nullable=True))
+
+    conn = op.get_bind()
+    nfs_shares = [dict(row) for row in conn.execute("SELECT * FROM sharing_nfs_share").fetchall()]
+    for entry in nfs_shares:
+        # use existing entry for first path and add new entries for subsequent paths
+        _id = entry.pop("id")
+        paths = eval(entry['nfs_paths'])
+        first_path = paths.pop(0)
```

Comment:
Just for good measure, do you think we should maybe handle this safely if for some reason the list is empty and migration fails ?